server/src/main/resources/db/changelogs/db.changelog-1.1.xml

```diff
@@ -0,0 +1,65 @@
+<?xml version="1.0" encoding="UTF-8"?>
+
+<databaseChangeLog
+  xmlns="http://www.liquibase.org/xml/ns/dbchangelog"
+  xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+  xsi:schemaLocation="http://www.liquibase.org/xml/ns/dbchangelog
+         http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-3.1.xsd">
+  <property name="now" value="now()" dbms="postgresql"/>
+
+
+  <changeSet author="pahuang@redhat.com" id="1">
+    <comment>create Account table</comment>
+    <createTable tableName="account">
+      <column name="id" type="SERIAL" autoIncrement="true">
+        <constraints primaryKey="true" />
+      </column>
+      <column name="name" type="varchar(80)" />
+      <column name="email" type="varchar(100)">
```

Comment:
We should avoid using arbitrary values for size. Max email size is 254 characters.

Reply:
done